### DISCUSSION TITLE
IR bands J,H,Ks are not converging

### DISCUSSION THREAD

**Discussion by ghost** [mboquien/cigale#76] [Q&A]
Hi! I need help in figuring out what is wrong with my SED. So I have both HSC (optical) and VISTA(IR)data that I have used to get my SED. The HSC data seems to work just fine. But my IR data is not converging I am not sure what is wrong. I have attached SED for one of my input, the pcigale.ini file and my catalog file consisting of 5 inputs. I also had one doubt in the configuration file, after running the pcigale genconf the default values that I get aren't they enough, how do I know what are the parameters that I have to give for different modules?(like sfhdelayed, bc03 etc)
[J000940.25+005519.2_best_model (1).pdf](https://github.com/mboquien/cigale/files/13826500/J000940.25%2B005519.2_best_model.1.pdf)

[Catalog-new.txt](https://github.com/mboquien/cigale/files/13826497/Catalog-new.txt)
[pcigale.ini.txt](https://github.com/mboquien/cigale/files/13826516/pcigale.ini.txt)
Thank you for the help!!!

**Comment by ghost** [comment]
I realised I made one mistake, I had my flux values in microJy and not mJy. So I changed that and ran the code again and I realised there is definitely something wrong with the configuration file. I would have to change the parameters. But if I do add more values for the parameters in each module the run time is too long more than an hour. Is this normal or am I doing something wrong? please help me out!!!
(fyi I am using google colab)

**Reply by mboquien** [reply]
It is normal and expected that the more models are computed, the longer the run takes. However in this specific case, the difference in the observed fluxes is such that I think there is an issue with the photometry.
For the `pcigale.ini` file, there is no need to set multiple values for `sfr_A` as `normalise` is set to `True`. as for the rest, I think there is very few models. I would not recommend using Google Collab, in particular if you are using the public version, the computing power is very limite. It should work fine on a regular laptop.

**Reply by ghost** [reply]
Hi! Thank you for writing back. Yes I realised there were some issues in fetching the aperture counts. I kind of fixed that and converted them to fluxes and plotted them. But my redshift seems to be too off. (I have added the IR data to the optical data already given to me, hence I have an idea of the z_abs value). Regarding the google colab, I tried working on the terminal but there seems to be some issue with the sqlite3 package. I tried fixing it but it just wasn't working. Google colab takes about 15-30 mins for more than 1000000 models, so seems to work for now. As of now I have only 270 objects so it isn't a lot of time. I have attached new values and the plots. Do you still think there is an issue with the photometry or am i doing something wrong. 
[Catalog-new.txt](https://github.com/mboquien/cigale/files/13905782/Catalog-new.txt)
[J001306.61-003020.7_best_model.pdf](https://github.com/mboquien/cigale/files/13905788/J001306.61-003020.7_best_model.pdf)
[J000913.70+002216.7_best_model (2).pdf](https://github.com/mboquien/cigale/files/13905789/J000913.70%2B002216.7_best_model.2.pdf)
Thanks for the help!!

**Reply by ghost** [reply]
And sorry for the dumb questions, but because I am completely new to this I am a little lost. How do I know what parameters I am to assign to my modules, it definitely cannot be trial and error, right? And how do I decide if my value for the flux is an upper limit, is there a threshold that I am missing out on to determine which flux value is termed as an upper limit?
